python/triqs_dft_tools/sumk_dft_transport.py

```diff
@@ -400,6 +405,68 @@ def _commutator(A, B):
 
 # ----------------- transport -----------------------
 
+def raman_vertex(sumk,ik,direction,code,isp, options=None):
+    r"""
+    Reads all necessary quantities for transport calculations from transport subgroup of the hdf5 archive.
+    Performs checks on input. Uses interpolation if code=wannier90.
+
+    Parameters
+    ----------
+    sum_k : sum_k object
+            triqs SumkDFT object
+    ik : integer
+            the ik index in which will be calculate the raman vertex
+    direction : string
+            direction to calculate the Raman vertex. 
+    code : string
+        DFT code from which velocities are being read. Options: 'wien2k', 'wannier90'
+    isp : integer
+            spin index
+    options : dictionary, optional
+        additional keywords necessary in case a new direction wants to be implemented
+
+    Returns
+    -------
+    raman_vertex : 2D array
+            numpy array size [n_orb, n_orb] of complex type
+    """
+    if code in ('wien2k'):
+        assert 0, 'Raman for wien2k not yet implemented' 
+    dir_names=['xx','yy','zz','B2g','B1g','A1g','Eg']
+    dir_array=[ [[1.,0.,0.],[0.,0.,0.],[0.,0.,0.]],
```

Comment:
```suggestion
    dir_array=np.array([ [[1.,0.,0.],[0.,0.,0.],[0.,0.,0.]], 
```
this should make l 457 obsolete?